### DISCUSSION TITLE
Не могу поставить лайк ни под одним постом

### DISCUSSION THREAD

**Discussion by dimonchik064** [OpenVK/openvk#1136] [Q&A] [answered]
Здравствуйте! Я Дмитрий Бесфамильный, тот самый пользователь OpenVK (https://ovk.to/dima2008). Несколько дней тому назад я столкнулся с серьезной проблемой - я не могу поставить лайк ни под одним постом.

Например, понравился мне какой-нибудь пост и я хочу поставить лайк. Ставлю лайк как обычно, ставится. Потом обновляю страницу и... лайка нет! Сколько раз я не пытался ставить лайк и обновлять страницу, но НИЧЕГО НЕ ПОМОГАЕТ! Эта проблема и в мобильной версии и в версии для ПК. Скажу честно - лайки я не снимал, они сами снимались после обновления страницы! Пожалуйста, дорогая команда OpenVK, исправьте эту проблему. Не ленитесь! Не бросайте пользователей OpenVK в топку! Я вас прошу - исправьте как-нибудь эту проблему! Пожалуйста!🥺🙏

Надеюсь, я буду услышан командой OpenVK и проблема будет исправлена в ближайшее время.

**Comment by veselcraft** [comment, marked as answer]
Во-первых, вы не туда зарепортили баг. Это нужно делать [сюда](https://github.com/OpenVK/openvk/issues)
Во-вторых, этот баг у нас не повторяется. Несмотря на тонну текста, никакой полезной информации помимо того, что кнопка лайка магическим образом не работает, нет. Ни браузера, ни её версии, ни конкретных примеров, ни видео демонстрации.
